parmed/tools/simulations/sanderapi.py

```diff
@@ -94,3 +98,39 @@ def energy(parm, args, output=sys.stdout):
         elif e.hbond != 0:
             output.write('     EHbond   = %20.7f' % e.hbond)
         output.write('\nTOTAL    = %20.7f\n' % e.tot)
+
+def minimize(parm, igb, saltcon, cutoff, tol, maxcyc):
+    """ Minimizes a snapshot. Use the existing System if it exists """
+    if not HAS_SANDER:
+        raise SimulationError('Could not import sander')
+    if not HAS_SCIPY:
+        raise SimulationError('Could not import scipy')
+
+    if parm.box is None:
+        if not igb in (0, 1, 2, 5, 6, 7, 8):
```

Comment:
I think those *used* to exist, but were deprecated and removed since they sucked.